tests/population/test_base_population.py

```diff
@@ -259,13 +205,13 @@ def test__assign_demography_with_initial_age(config, include_sex):
     _check_population(simulants, initial_age, step_size, include_sex)
 
 
-def test__assign_demography_with_initial_age_zero(config, include_sex):
+def test__assign_demography_with_initial_age_zero(make_base_simulants, config, include_sex):
     pop_data = dt.assign_demographic_proportions(
         make_uniform_pop_data(age_bin_midpoint=True),
         include_sex=include_sex,
     )
     pop_data = pop_data[pop_data.year_start == 1990]
-    simulants = make_base_simulants()
+    simulants = make_base_simulants
```

Comment:
Nit: you don't need to reassign this to `simulants`. Same thing in several other test cases